sound/hybrid_font.h

```diff
@@ -193,7 +202,14 @@ class HybridFont : public SaberBase {
   void SB_EndLockup() override {
     if (lock_player_) {
       // Polyphonic case
-      lock_player_->set_fade_time(0.3);
+      if (endlck.files_found()) { // polyphonic end lock
+        // if playing an end lock fade the lockup faster
+        lock_player_->set_fade_time(0.003);
+        PlayPolyphonic(&endlck);
```

Comment:
might be a good idea to fall back to the slow fadeout if PlayPolyphonic fails to get a wav player

Reply:
Good idea. I'll add that

Reply:
Simplified the if/else to achieve same effect.